ui/src/app/components/gke-app-configuration-panels/create-gke-app-button.tsx

```diff
@@ -1,43 +1,60 @@
 import * as React from 'react';
 import { useState } from 'react';
 
-import { CreateAppRequest, UserAppEnvironment } from 'generated/fetch';
+import { AppType, CreateAppRequest, UserAppEnvironment } from 'generated/fetch';
 
 import { canCreateApp } from 'app/components/apps-panel/utils';
 import { Button } from 'app/components/buttons';
 import { TooltipTrigger } from 'app/components/popups';
+import { SUPPORT_EMAIL } from 'app/components/support';
 import { ApiErrorResponse, fetchWithErrorModal } from 'app/utils/errors';
+import { NotificationStore } from 'app/utils/stores';
 import { appTypeToString, createUserApp } from 'app/utils/user-apps-utils';
 
 export interface CreateGKEAppButtonProps {
   createAppRequest: CreateAppRequest;
   existingApp: UserAppEnvironment | null | undefined;
   workspaceNamespace: string;
   onDismiss: () => void;
+  username: string;
 }
 
 export function CreateGkeAppButton({
   createAppRequest,
   existingApp,
   workspaceNamespace,
   onDismiss,
+  username,
 }: CreateGKEAppButtonProps) {
   const [creatingApp, setCreatingApp] = useState(false);
   const createEnabled = !creatingApp && canCreateApp(existingApp);
 
   const appTypeString = appTypeToString[createAppRequest.appType];
 
+  const conflictError: NotificationStore = {
+    title: `Error Creating ${appTypeString} Environment`,
+    message: `Please wait a few minutes and try to create your ${appTypeString} Environment again.`,
+    onDismiss,
+  };
+
+  const sasAuthError: NotificationStore = {
+    title: 'Error Creating SAS Environment',
+    message:
+      `The apps server has indicated that the user ${username} is not authorized to create SAS apps. ` +
```

Comment:
good question.  Maybe "app" is also not quite right because we're using "environment" in many places.

Could also simply this further with "the server" or maybe leave that part out completely: "The user [username] is not authorized"

Reply:
updated